cap/modules/fixtures/schemas/cms-questionnaire.json

```diff
@@ -1,1370 +1,1746 @@
 {
-  "name": "cms-questionnaire",
-  "version": "0.0.1",
-  "fullname": "Statistics Questionnaire",
-  "experiment": "CMS",
-  "is_indexed": true,
-  "use_deposit_as_record": true,
-  "deposit_schema": {
-    "additionalProperties": false,
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "type": "object",
-    "properties": {
-      "other_stats": {
-        "properties": {
-          "blind_analysis": {
-            "type": "object",
-            "properties": {
-              "name": {
-                "enum": ["no_no_practical_way", "no", "yes"],
-                "type": "string",
-                "description":
-                  "Do you perform a blind analysis? If so, can you succinctly describe what was done?",
-                "title": "Blind Analysis"
+  "name":"cms-questionnaire",
+  "version":"0.0.1",
+  "fullname":"CMS Statistics Questionnaire",
+  "experiment":"CMS",
+  "is_indexed":true,
+  "use_deposit_as_record":true,
+  "deposit_schema":{
+    "additionalProperties":true,
+    "$schema":"http://json-schema.org/draft-04/schema#",
+    "type":"object",
+    "properties":{
+      "_user_edited": {
+        "type": "boolean"
+      },
+      "_deposit": {
+        "$ref": "https://analysispreservation.cern.ch/schemas/deposits/records/deposit-v0.0.1.json"
+      },
+      "_buckets": {
+        "$ref": "https://analysispreservation.cern.ch/schemas/deposits/records/buckets-v0.0.1.json"
+      },
+      "_experiment": {
+        "type": "string"
+      },
+      "_access": {
+        "$ref":
+          "https://analysispreservation.cern.ch/schemas/deposits/records/access-v0.0.1.json"
+      },
+      "$ana_type": {
+        "type": "string"
+      },
+      "$schema": {
+        "type": "string"
+      },
+      "general_title": {
+        "type": "string"
+      },
+      "analysis_context":{
+        "type":"object",
+        "properties":{
+          "name":{
+            "type":"string",
+            "description":"Please insert the name of the person filling up the questionnaire.",
+            "title":"1.1 Your name"
+          },
+          "email":{
+            "type":"string",
+            "title":"1.2 Your email address"
+          },
+          "wg":{
+            "type":"string",
+            "enum":[
+              "B physics and quarkonia (BPH)",
+              "Future (FTR)",
+              "Beyond-two-generations (B2G)",
+              "Exotica (EXO)",
+              "Higgs physics (HIG)",
+              "Heavy ions physics (HIN)",
+              "Standard Model physics (SMP)",
+              "Supersymmetry (SUS)",
+              "Top physics (TOP)",
+              "Physics of the CMS-Totem Precision Proton Spectometer (PPS)",
+              "Other (please specify)"
+            ],
+            "description":"PAG/POG identifier",
+            "title":"1.3. Working group"
+          },
+          "cadi_id":{
```

Comment:
Lets add it like here https://github.com/cernanalysispreservation/analysispreservation.cern.ch/blob/master/cap/modules/fixtures/schemas/cms.json#L136